indy/Dockerfile

```diff
@@ -59,11 +46,16 @@ RUN chgrp -R 0 /opt && \
     chmod -R g=u /usr/share/indy && \
     chgrp -R 0 /home/indy && \
     chmod -R g=u /home/indy && \
-    chown -R 1001:0 /home/indy && \
-    chmod 644 /etc/profile.d/setup-user.sh
+    chown -R 1001:0 /home/indy
+
+EXPOSE 8080 8081 8000
 
 USER 1001
```

Comment:
So just curious that where is user 1001 is created and mapped?

Reply:
it doesn't matter, this is just for OpenShift arbitary user execution, I tested this on GPC it works with this line and without user wrapper script